docs/layouts/shortcodes/get-started/connect-service.md

```diff
@@ -0,0 +1,66 @@
+## Connect database to app
+
+Now connect the database to your app.
+
+<div x-show="stack === 'php' || stack === 'nodejs' || stack === 'python'">
+
+First, add the Platform.sh Config Reader library to make the connection easier.
+
+<div x-show="stack === 'php'">
+
+```bash
+composer require platformsh/config-reader
+```
+
+</div>
+
+<div x-show="stack === 'nodejs'">
+
+```bash
+npm install platformsh-config
+```
+
+</div>
+
+<div x-show="stack === 'python'">
+  
+```bash
+pip install platformshconfig
+```
+
+</div>
+
+Then connect to the database in your app using the library.
+You can choose where to do this that makes sense in your app.
```

Comment:
```suggestion
You can choose where to do this, depending on what makes sense for your app.
```

Reply:
I like it much better, just without the comma.